leapp/workflows/__init__.py

```diff
@@ -77,12 +84,25 @@ def __init__(self, logger=None, auto_reboot=False):
         self._experimental_whitelist = set()
         self._auto_reboot = auto_reboot
 
+        if self.configuration:
+            config_actors = [actor for actor in self.tag.actors if self.configuration in actor.produces]
+            if config_actors and len(config_actors) == 1:
+                self._phase_actors.append((
+                    _ConfigPhase,
+                    PhaseActors((), 'Before'),
+                    PhaseActors(tuple(config_actors), 'Main'),
+                    PhaseActors((), 'After')))
+
```

Comment:
Missing the `elif` statement. It should fail when there are more actors producing the message.

Reply:
Done